src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -400,16 +401,18 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                     }
                     else if (value is IDictionary<string, object>[])
                     {
-                        value = ((IEnumerable<IDictionary<string, object>>)value).Select(p => NormalizeBindingData(p)).ToArray();
+                        value = ((IEnumerable<IDictionary<string, object>>)value)
+                            .Select(p => NormalizeBindingData(p)).ToArray();
                     }
-                    else
+                    else if (value is IDictionary<string, string>)
                     {
-                        if (!IsEdgeSupportedType(type))
-                        {
-                            // for values not supported by edge, we just
-                            // stringify the value
-                            value = value.ToString();
```

Comment:
It's safe to remove this because all of the existing binding data for all our triggers were simple types like strings, etc. The one exception was EventHub trigger, which has a PartitionContext property as part of its binding data. Previously we would to string that (not very useful - just the type name). Now, we expand it out into its useful poco properties as you can see in the tests.